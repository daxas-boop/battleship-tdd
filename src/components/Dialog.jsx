import React from 'react';
import styled from '@emotion/styled';

const DialogContainer = styled.section `
    display:flex;
    margin:20px;
`

const IconLink = styled.a `
    text-align:center;
    margin-top:20px;
`

const Dialog = () => {
    return(
            <DialogContainer class="message -left">
                <i class="nes-bcrikko" style={{alignSelf: 'flex-end'}}></i>
                <section class="nes-balloon from-left is-dark" style={{width:'75%', marginLeft:30}} >
                    <p>This game was created by daxas-boop.</p>
                    <IconLink href='https://github.com/daxas-boop/' target='_blank' rel="noopener noreferrer">
                        <i className='nes-icon github nes-pointer is-medium'></i>
                    </IconLink>
                </section>
            </DialogContainer>
    )
}

export default Dialog;