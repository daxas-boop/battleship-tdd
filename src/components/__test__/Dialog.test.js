import React from 'react';
import renderer from 'react-test-renderer';
import Dialog from '../Dialog';

test('renders correctly', () => {
    const component = renderer.create(<Dialog></Dialog>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})