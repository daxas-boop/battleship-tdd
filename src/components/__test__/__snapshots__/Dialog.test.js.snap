// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="message -left css-x0ow50"
>
  <i
    className="nes-bcrikko"
    style={
      Object {
        "alignSelf": "flex-end",
      }
    }
  />
  <section
    className="nes-balloon from-left is-dark"
    style={
      Object {
        "marginLeft": 30,
        "width": "75%",
      }
    }
  >
    <p>
      This game was created by daxas-boop.
    </p>
    <a
      className="css-1hfqtwh"
      href="https://github.com/daxas-boop/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <i
        className="nes-icon github nes-pointer is-medium"
      />
    </a>
  </section>
</section>
`;
